README.md

```diff
@@ -10,15 +10,15 @@ IPFX is a Python package for computing intrinsic cell features from electrophysi
 
 This software is designed for use in the Allen Institute for Brain Science electrophysiology data processing pipeline.
 
-For usage and installation instructions, see the [documentation](https://ipfx.readthedocs.io/en/latest//).
+For usage and installation instructions, see the [documentation](https://ipfx.readthedocs.io/en/latest/).
 
 Quick Start
 ------------
-To start analyzing data now, check out the [quick_start](docs/quick_start.rst) . For a more in depth guide to IPFX, see [tutorial](docs/tutorial.rst) 
+To start analyzing data now, check out the [quick_start](https://ipfx.readthedocs.io/en/latest/quick_start.html) . For a more in depth guide to IPFX, see [tutorial](https://ipfx.readthedocs.io/en/latest/tutorial.html)
 
 Contributing
 ------------
-We welcome contributions! Please see our [contribution guide](CONTRIBUTING.md) for more information. Thank you!
+We welcome contributions! Please see our [contribution guide](https://github.com/AllenInstitute/ipfx/blob/master/CONTRIBUTING.md) for more information. Thank you!
 
```

Comment:
why did you change the links to point to the master branch? This is confusing because when clicking on links you will see content belonging to different branches.

Reply:
This was merged from the master branch. If I change it to this branch, after merged, will confuse other branches?

Reply:
you should rebase you work on top of the release branch and target the release branch with our PR